platform/src/main/resources/log4j2-spring.yaml

```diff
@@ -53,12 +53,4 @@ Configutation:
       level: info
       AppenderRef:
         - ref: ${env:LOG_APPENDER:-console_appender}
-    Logger:
```

Comment:
What are the log levels set to after removing this?

Reply:
Default is always INFO